tools/c-parser/README.md

```diff
@@ -7,31 +8,125 @@
 The StrictC translation tool
 ============================
 
-To install, see the file INSTALL in the `src/c-parser` directory.
+The StrictC translation tool, also called the Isabelle C parser, translates a
+large subset of C99 code into the imperative language [Simpl] embedded in the
+theorem prover [Isabelle/HOL]. The Simpl language provides a verified
+verification condition generator and further tools for software verification.
+The tools is aimed at Isabelle/HOL experts with knowledge in program
```

Comment:
Unclear if this should be `The tool is` or `The tools are`

Reply:
The tool is. Thanks for spotting that, will fix.